### PR TITLE
Resolves #40: Add /help command.

### LINKED ISSUE #40: Add /help command
I was very confused with some errors when trying to run the bot, and then realized that it doesn't have a `/start` command. Having a `/help` to see available commands could improve the bot's usability a lot.

### PR DESCRIPTION
**Pull request by nicoschmdt**
Adds a simple help command with a summary of available commands

### CODE REVIEW

File: karmagrambot/commands.py
Comment (JPTIZ) [inline karmagrambot/commands.py]:
```suggestion
    response = dedent(f"""
                Karma is obtained through replying messages with + or -

                The following commands are available:

                \\average_length - informs your average message length.
                \\karma - informs your respective karma.
                \\karmas - shows the top 10 karmas in a given group.
                \\devil - replies the user informing who is the person with the lowest karma.
                \\saint - replies the user informing who is the person with the highest karma.
                """)
```

And then remember to add on top of the file:

```python
from textwrap import dedent
```

**Review by JPTIZ — APPROVED** [review]
Approved for hacktoberfest. Waiting for further review from others.

**Review by GabrielSimonetto — APPROVED** [review]
That looks great! (:

**Pull request merged**
